hotel/__openerp__.py

```diff
@@ -0,0 +1,32 @@
+# -*- encoding: utf-8 -*-
+
+{
+    "name": "Hotel Management Base",
+    "version": "1.0",
```

Comment:
Please check the guidelines on how to define the module version number.
https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers